core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -122,6 +122,7 @@ private[spark] abstract class BasePythonRunner[IN, OUT](
   protected val authSocketTimeout = conf.get(PYTHON_AUTH_SOCKET_TIMEOUT)
   private val reuseWorker = conf.get(PYTHON_WORKER_REUSE)
   protected val faultHandlerEnabled: Boolean = conf.get(PYTHON_WORKER_FAULTHANLDER_ENABLED)
+  protected val hideTraceback: Boolean = false
```

Comment:
Can we use `conf.get(PYSPARK_HIDE_TRACEBACK)` here so that we don't need to override every subclass?

Reply:
The config is defined in `org.apache.spark.sql.internal.SQLConf` which seems to be inaccessible from here. `PYSPARK_SIMPLIFIED_TRACEBACK` is also defined in `SQLConf` so `BasePythonRunner` subclasses have to override it.

Is there an advantage for putting it in `SQLConf` rather than e.g. `org.apache.spark.internal.config.Python`?

Reply:
The conf in `SQLConf` is session-based conf that also can be set in runtime, and any conf in `core` module or `StaticSQLConf` is cluster-wide conf and can't be changed while the cluster is running.